test/java/seedu/addressbook/storage/StorageFileTest.java

```diff
@@ -25,7 +25,8 @@
 import static seedu.addressbook.util.TestUtil.assertFileDoesNotExist;
 
 public class StorageFileTest {
-    private static final String TEST_DATA_FOLDER = "test/data/StorageFileTest";
+    private static final String TEST_DATA_FOLDER = "C:/Users/chungming/Google Drive/" +
```

Comment:
Hard-coding the path is not encouraged as other developers that work with you will not be able to run the code.